pkg/controller/migcluster/migcluster_controller.go

```diff
@@ -37,7 +37,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
-var log = logf.Log.WithName("controller")
+var log = logf.Log.WithName("cluster")
```

Comment:
I'm not convinced it adds value since it's unlikely that other loggers will be logging in our manager process.  The cost associated is adding 3 chars to every message.

Reply:
As we discussed, I agree with you after taking out some additional log messages.